### ISSUE TITLE
Be more judicious with -webkit-font-smoothing

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/brand-patterns#22]
Currently, `-webkit-font-smoothing: antialiased;` is applied to the `body`. This works great for light text on a dark background; however, body text on the page looks really strange when this is applied.

  > So here is yet another plea for designers to stop “fixing” WebKit font smoothing by disabling subpixel rendering. Feel free to use it on light text on dark backgrounds, feel free to use it to fix custom font rendering on Windows or to style specific bits of text on the page to make it look more slender, but for main portions of text where readability is paramount please leave the default setting alone and let the operating system handle the smoothing. -[Source](http://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/)

You can see the difference between `antialiased` and `auto` specifically on the "Fieldsets and Form Elements" sections of http://patterns.wvu.edu/patterns . 

Long story short, consider applying `antialiased` anywhere there's light text on a dark background but not globally for every single element on the page.
